### PR TITLE
INT-106: Add payment service

### PR DESCRIPTION
**Pull request by yuliia-bondareva-corefy**

### CODE REVIEW

File: data/payment_providers.json [hunk 1]
```diff
@@ -19593,5 +19593,18 @@
     "payment_method" : [
       "payment_card"
     ]
+  },
+  {
+    "status": "active",
+    "code": "kluwp",
+    "vendor": "kluwp",
+    "payout_method": [
```
Comment (TonyTramb) [inline data/payment_providers.json:19601]: payment_method

File: data/payment_services.json [hunk 2]
```diff
@@ -401,6 +401,15 @@
...
     "amount_max": 100000
   },
+  {
+    "status": "active",
+    "code": "payment_card_hrk_hpp",
+    "currency": "HRK",
+    "method": "payment_card",
+    "flow": "hpp",
+    "amount_min": 0.01,
+    "amount_max": 100000
```
Comment (TonyTramb) [inline data/payment_services.json:411]: image

Comment (yuliia-bondareva-corefy) [reply data/payment_services.json]: change amount_max

File: data/payment_providers.json (hunk 1 above)
Comment (yuliia-bondareva-corefy) [reply data/payment_providers.json]: change method

**Pull request closed without merging**